docker/Dockerfile-scalefield

```diff
@@ -9,10 +9,14 @@ ENV GIT_TIME=${GIT_TIME}
 ADD pgwatch2 /pgwatch2
 RUN cd /pgwatch2 && bash build_gatherer.sh
 
-
-FROM ubuntu:22.04
-
-RUN apt-get -q update && apt-get -qy install wget git python3 python3-psycopg2 python3-yaml && apt autoremove -y && mkdir /pgwatch2
+FROM alpine
+
+RUN apk update && \
+    apk add --no-cache wget git python3 py3-psycopg2 py3-yaml && \
```

Comment:
in [pgwatch3](https://github.com/cybertec-postgresql/pgwatch3/blob/master/docker/Dockerfile) it will be even more tiny :)

Reply:
I've removed and tested withtout wget. Definitly not needed. But in current version we need python in order to execute rollout_helper.py. 
It assume it could be optimised but we are running outside of scope of issue that we'r fixing.

Reply:
Ah, I see... Indeed... rollout helpers... 🤦